src/prebid.js

```diff
@@ -230,7 +230,7 @@ $$PREBID_GLOBAL$$.setTargetingForGPTAsync = function (adUnit) {
   targeting.setTargeting(targetingSet);
 
   // emit event
-  events.emit(SET_TARGETING);
+  events.emit(SET_TARGETING, adUnit);
```

Comment:
adUnit is optional here. Probably better to define some structure. Is this an array, object etc?

Reply:
Once that is done, it should be applied to the other SET_TARGETING emitter - see line 250

Reply:
Just to be clear, my preference is:

events.emit(SET_TARGETING, { adUnits: [adUnit1, adUnitN...]});

Reply:
Thanks for the feedback. Will fix!